bin/common.ml

```diff
@@ -938,22 +931,20 @@ module Builder = struct
         & info
             [ "error-reporting" ]
             ~doc:
-              "Controls when the build errors are reported.\n\
-               $(b,early) - report errors as soon as they are discovered.\n\
-               $(b,deterministic) - report errors at the end of the build in a \
-               deterministic order.\n\
-               $(b,twice) - report each error twice: once as soon as the error is \
-               discovered and then again at the end of the build, in a deterministic \
-               order.")
+              "Controls when the build errors are reported. $(b,early) - report errors \
```

Comment:
This is meant to be a list of bullets.

Reply:
But in fact it is not formatted as a list of bullets in `dune build --watch`. I think it is better to let the string be auto-formatted instead of giving the impression that we have the ability to do formatting ourselves.

Reply:
In that case, let's remove the `-` and just list the values in sentences.

Reply:
Done.